images/airflow_Dockerfile.sle_15sp1

```diff
@@ -0,0 +1,150 @@
+# Copyright 2018 AT&T Intellectual Property.  All other rights reserved.
```

Comment:
We need kiwi files that are build in IBS to follow SUSE guidelines and to have maintainable containers. And we are currently in the process to create a workflow (cc @stefannica , @JanZerebecki ) to create containers for socok8s.

Reply:
This is first step to get sle support working ( as per this task SOC-9928 )  and we can change it going further.

Reply:
ok. thanks for the hint